setup.py

```diff
@@ -303,14 +340,30 @@ def objects_to_spec(spec_object: SpecObject, imports: str, fork: str, ordered_cl
             + '\n\n' + f"fork = \'{fork}\'\n"
             + '\n\n' + new_type_definitions
             + '\n' + SUNDRY_CONSTANTS_FUNCTIONS
-            + '\n\n' + constants_spec
+    )
+
+    if fork == 'altair':
+        altair_ssz_dep_constants = '\n'.join(map(lambda x: '%s = %s' % (x, ALTAIR_HARDCODED_SSZ_DEP_CONSTANTS[x]), ALTAIR_HARDCODED_SSZ_DEP_CONSTANTS))
+        spec += (
+            ALTAIR_SUNDRY_FUNCTIONS
+            + '\n\n' + altair_ssz_dep_constants
+        )
+
+    spec += (
+            '\n\n' + constants_spec
             + '\n\n' + CONFIG_LOADER
             + '\n\n' + ordered_class_objects_spec
             + '\n\n' + functions_spec
             + '\n' + PHASE0_SUNDRY_FUNCTIONS
     )
+
     if fork == 'phase1':
         spec += '\n' + PHASE1_SUNDRY_FUNCTIONS
+
+    if fork == 'altair':
```

Comment:
Should these statements just go into the `if fork == 'altair'` conditional above? Or is there a reason to have two `if fork == 'altair'`conditionals?

Reply:
The reason was that `altair_ssz_dep_constants` has to be defined before `ordered_class_objects_spec`. And `altair_ssz_dep_constants_verification` has to be defined after `ordered_class_objects_spec`.

I just added a commit to refactor this section. Hope that it's more readable now.